### DISCUSSION TITLE
Problems with mutate and input[type="datetime-local"]

### DISCUSSION THREAD

**Discussion by dudusotero** [vercel/swr#526] [General]
Hello everyone! I'm here to show a quick problem I have at this moment.

Why when I submit a form and do a mutate with the new form values, all input with type "datetime-local" changes to another value?

I made a quick example of my problem, so everyone can check without do any code: [link here](https://stackblitz.com/edit/react-kpf1yh)

Gif with the occurring problem:
![ezgif-3-b37182d6428e](https://user-images.githubusercontent.com/7073199/87592555-752b1680-c6c0-11ea-9871-20302026430d.gif)

**Comment by sergiodxa** [comment]
The problem is that every time you update the cached value the `post` object is changed with a new one (same values, different reference), that triggers your effect where you update the form and create a new date.

**Reply by dudusotero** [reply]
@sergiodxa when the effect triggers, my ```post.date``` value, already changed. The problem occurs before setting a new Date.
:(

I don't have a solution for that problem yet, do you have some ideas of what I need to do to persist the value of ```post.date``` between mutations?
